### DISCUSSION TITLE
Welcome to aptpac Discussions!

### DISCUSSION THREAD

**Discussion by Itai-Nelken** [Itai-Nelken/aptpac#6] [General]
## 👋 Welcome!

You can post here anything that is related to `aptpac` but isn't a issue.
Have fun!

**Comment by CleanMachine1** [comment]
Sorry I haven't been keeping up with aptpac, I have started to help over at tldr-pages.
However I have pinned aptpac on my profile for you :)

**Reply by Itai-Nelken** [reply]
Thanks!
